examples/node-pub

```diff
@@ -13,12 +15,10 @@
  * limitations under the License.
  */
 
-#!/usr/bin/env node
-
 /* jslint node: true */
 'use strict';
 
-const nats = require('nats');
+const nats = require('../');
```

Comment:
I don't understand this part. I assume that since these examples are in `examples/` directory, the `../` directive seem to point to root dir, but...